packages/std/src/math/decimal256.rs

```diff
@@ -364,6 +372,15 @@ impl ops::DivAssign<Uint256> for Decimal256 {
     }
 }
 
+impl<A> std::iter::Sum<A> for Decimal256
```

Comment:
I really like the "if two things of type A can be added to get a Decimal, an iterator of those can be summed" :+1:

Reply:
Yeah, that's cool.

I was super sonfused about `Decimal256: ops::Add<A, Output = Decimal256>,`, which specifies `A`, not `Decimal256`.